Source/hydro/Castro_ctu_1d.F90

```diff
@@ -354,7 +354,7 @@ subroutine umeth(q, q_lo, q_hi, &
 #endif
                 qaux, qa_lo, qa_hi, &
                 shk, shk_lo, shk_hi, &
-                1, lo(1), hi(1)+1, 0, 0, 0, 0, 0, &
+                1, lo, hi+1, &
```

Comment:
Needs to be hi+dg I think, not hi+1

Reply:
yes :)

Reply:
in my defense, I am working on getting rid of this function entirely in a different branch.